pom.xml

```diff
@@ -140,7 +140,7 @@
                         </executions>
                         <configuration>
                             <excludes>
-                                <exclude>**/*RtYamlStreamBuilder$*.*</exclude>
+                                <exclude>**/*YamlStream.java</exclude>
```

Comment:
We exclude this interface from code-coverage calculation in order to avoid unit testing the default Stream methods which are just delegations.